sdk/app_cpu1/common/drv/timing_manager.h

```diff
@@ -13,12 +13,19 @@
 #define ISR0_PRIORITY            (0x08)
 #define ISR_RISING_EDGE          (0x3)
 #define DEFAULT_SENSOR_ENABLE    (0x0000)
-#define DEFAULT_PWM_RATIO        (0x1)
+#define DEFAULT_PWM_RATIO        (10)
 #define LOWER_16_MASK            (0x0000FFFF)
 #define NUM_SENSORS              (10)
 #define TIMING_MANAGER_BASE_ADDR (XPAR_AMDC_TIMING_MANAGER_0_S00_AXI_BASEADDR)
 #define UPPER_16_SHIFT           (16)
 
+// Expected times (us) for all the sensors based on default values
+#define ADC_DEFAULT_TIME        (0.82)
+#define ENCODER_DEFAULT_TIME    (0.005)
+#define AMDS_DEFAULT_TIME       (11.8)
+#define EDDY_DEFAULT_TIME       (4.11)
+#define MAX_DEFAULT_SENSOR_TIME (AMDS_DEFAULT_TIME)
```

Comment:
Maybe I missed something, what is the purpose of declaring these expected times in the code? Are they used somewhere, or just for the curious user to discover?

Reply:
This was something Nathan and I talked about - up above where I talk about the scheduler's time quantum checking, once you enable a sensor the ISR is called super fast and the quantum check triggers, so I use the max sensor time as a condition for that in `scheduler.c`

Reply:
ah I see, thanks!